src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -109,6 +111,7 @@ public static TypedData ToRpc(this object value)
                             var bytes = new byte[length];
                             request.Body.Read(bytes, 0, length);
                             body = bytes;
+                            rawBody = bytes.ToString();
```

Comment:
This will give you the type name. Use `Encoding.UTF8.GetString` (or appropriate encoding)

Reply:
May want to make sure you have a failing test for this before fixing :)